colour_runner/django_runner.py

```diff
@@ -4,9 +4,14 @@
 class ColourRunnerMixin(object):
     test_runner = ColourTextTestRunner
 
+    def __init__(self, *args, **kwargs):
+        self.no_colour = kwargs['no_color']
```

Comment:
This spelling inconsistency is annoying, but it allows us to avoid customising `manage.py test`.

Reply:
That's understandable. Thanks for the explanation.